config/environments/test.rb

```diff
@@ -12,7 +12,7 @@
   # Do not eager load code on boot. This avoids loading your whole application
```

Comment:
Should probably update the comment here to describe why we're setting this to true.